src/ray/common/ray_config_def.h

```diff
@@ -35,11 +35,11 @@ RAY_CONFIG(int64_t, ray_cookie, 0x5241590000000000)
 RAY_CONFIG(int64_t, handler_warning_timeout_ms, 1000)
 
 /// The duration between heartbeats sent by the raylets.
-RAY_CONFIG(int64_t, raylet_heartbeat_timeout_milliseconds, 1000)
+RAY_CONFIG(int64_t, raylet_heartbeat_timeout_milliseconds, 100)
```

Comment:
Thanks for reverting this. Btw this is a super huge change to Ray scheduling behavior, shouldn't these kind of decisions be discussed before merge?

cc @stephanie-wang